src/Onboarding/EulaModal.tsx

```diff
@@ -1,26 +1,24 @@
-import React, { useEffect, useState, FunctionComponent } from "react"
+import React, { useState, FunctionComponent } from "react"
 import {
   TouchableOpacity,
   Linking,
   Modal,
+  ActivityIndicator,
   StyleSheet,
   View,
   SafeAreaView,
   StatusBar,
   Image,
 } from "react-native"
 import { useTranslation } from "react-i18next"
-import loadLocalResource from "react-native-local-resource"
 import WebView, { WebViewNavigation } from "react-native-webview"
 import { SvgXml } from "react-native-svg"
 
 import { Button, RTLEnabledText } from "../components"
-import en from "../locales/eula/en.html"
-import es_PR from "../locales/eula/es_PR.html"
-import ht from "../locales/eula/ht.html"
```

Comment:
Should we also remove these files since now the EULA is being fetched from the web?